### ISSUE TITLE
Sending files from files and photos app to other devices

### ISSUE THREAD

**Issue by lukejd83** [williamMillington/warpinator-iOS#6]
Would it be an easy integration to allow sending photos or files from the iOS app to other devices?

**Comment by Glutnix** [comment]
yes, this would be my prime usecase for Warpinator - please add share sheet support!

**Comment by stefan123t** [comment]
My prime use would be to share photos from iPhone to Linux Mint.

Currently it only allows to Send Files from iOS to Linux.
